server/src/controller/user.controller.js

```diff
@@ -130,8 +133,20 @@ exports.querylogin = (req, res) => {
             res.json({
               success: true,
               _id: user._id,
-              token: "manju" + token
+              token: "%$%#&$" + token
```

Comment:
As discussed during class, adding your own complexity to a token hash will result in a no-op, and is bad practice.
Because you don't store this "unique string" in a config or env file, this complexity also loses it value, since your repository is open source. Anyone can see you add `%$%#&$` to your token.

Reply:
thanks for reminding me !! I am using unique string in a .env file, I will remove this (token: "%$%#&$" + token)  from final code.